### ISSUE TITLE
Sample list for download

### ISSUE THREAD

**Issue by andrewkrh** [StraboSpot/StraboSpot-Mobile#403]
Maybe this belongs on the "server" page.

**Describe the solution you'd like**
It would be helpful to be able to quickly download a list of samples collected in a recent project. This could be a similar feature to the download > Stereonet Mobile option online. List could include:
Sample Label, Location, Type, Description.

This is particularly useful in international research that requires paperwork be filed with local customs offices before samples can be shipped.

**Comment by jessicagood** [comment]
@andrewkrh Yes, this could be posted here https://github.com/StraboSpot/StraboSpot-Server/issues
@jasonash

**Comment by jasonash** [comment]
Hi @andrewkrh . I have added a samples download to the StraboSpot.org "My Data" page per your request above:

<img width="417" alt="Screen Shot 2022-02-08 at 9 00 15 AM" src="https://user-images.githubusercontent.com/7904402/153013988-fb15fea8-5df8-4673-83cc-0af42e9acfc6.png">

I hope this helps. Please let us know if you need anything else.
